_posts/2019-08-12-what-the-cluster.markdown

```diff
@@ -0,0 +1,75 @@
+---
+layout: post
+title:  "Snowflake: What the cluster"
+date:   2019-08-12
+tags: snowflake
+author: Natalie Laing
+image: img/snow.jpg
+---
+![Snowflake Unicorn](/img/blog/what-the-cluster/Uni-Snowflake.png)
+
+I recently attended a Snowflake 4 day partner bootcamp in Melbourne to get some hands on experience with Snowflake.
+
+* Snowflake is a data warehouse built for the cloud that can handle structured and semi-structured data. Snowflake supports an unlimited number of simultaneous users so multiple groups of users can access data at the same time with no performance degradation.
+* Snowflake is a zero-management data warehouse-as-a-service.
+* Snowflake deals with all the parts of setting up a data warehouse solution so you don’t have to; you get to play with all the bells and whistles to tune snowflake into your ideal data warehouse solution.
+* Snowflake deals with all the management and setting up of your data warehouse so you can focus on the processes you care about.
+* Snowflake comes with a fail safe which stores your historical data. This is non configurable and only accessible by contacting Snowflake support. Snowflake will take a copy of all tables, views, etc and store this for seven days and is only to be used to retrieve data in a case of damage or system failure. You get a seven day fail safe as default and this feature can’t be turned off.
```

Comment:
`tables, views, etc and store.` The `etc` reads weird. You can be specific or just take it out.